src/structure.ts

```diff
@@ -53,7 +53,7 @@ export interface Components<Ctx extends Context = Context, Req extends Request =
 /**
  * Validate that a given value is a {@link Command} object
  */
-const isCommand = (value: any, warn = false): value is Command => {
+const isCommand = (value: Command, warn = false): value is Command => {
```

Comment:
This type is definitely wrong -- this is a type-guard, the whole point is to given it an unknown value and it tells you if it is of the type -- typing it as the input already being the type makes it a no-op essentially